WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -42,8 +42,7 @@ struct NavigationItemRow: ImmuTableRow {
 
         WPStyleGuide.configureTableViewCell(cell)
 
-        cell.imageView?.tintColor = tintColor
-        cell.textLabel?.textColor = tintColor
+        cell.imageView?.tintColor = tintColor ?? .secondaryLabel
```

Comment:
- Reverting to the `.secondaryLabel` color if `tintColor` is not set (default from WPStyleGuide.configureTableViewCell(cell))
- No longer need `cell.textLabel?.textColor = tintColor` because I updated these button to use `ButtonRow`